Source/Lib/Codec/EbCodingLoop.c

```diff
@@ -2867,11 +2867,11 @@ void EncodePassPackLcu(
     if ((sequenceControlSetPtr->staticConfig.compressedTenBitFormat == 1))
     {
 
-        const EB_U32 inputLumaOffset = ((lcuOriginY + inputPicture->originY)       * inputPicture->strideY) + (lcuOriginX + inputPicture->originX);
-        const EB_U32 inputCbOffset = (((lcuOriginY + inputPicture->originY) >> 1)  * inputPicture->strideCb) + ((lcuOriginX + inputPicture->originX) >> 1);
-        const EB_U32 inputCrOffset = (((lcuOriginY + inputPicture->originY) >> 1)  * inputPicture->strideCr) + ((lcuOriginX + inputPicture->originX) >> 1);
+        const EB_U32 inputLumaOffset = ((lcuOriginY + inputPicture->originY)                      * inputPicture->strideY) + (lcuOriginX + inputPicture->originX);
```

Comment:
Remove extra spaces at line 2870?

Reply:
Hi @intelmark , could you also fix this one?

Reply:
Believe this is fixed.  Is there a line number you can reference?

Reply:
line 2869

Reply:
corrected..